source/base/function_signed_distance.cc

```diff
@@ -76,7 +76,7 @@ namespace Functions
       const SymmetricTensor<2, dim> hess =
         unit_symmetric_tensor<dim>() / distance -
         symmetrize(outer_product(center_to_point, center_to_point)) /
-          std::pow(distance, 3);
```

Comment:
You missed changing to `fixed_power` in `source/base/function_signed_distance.cc`